chronosphere/resource_drop_rule.go

```diff
@@ -65,10 +68,19 @@ func (dropRuleConverter) toModel(
 	if err != nil {
 		return nil, err
 	}
+
+	// Mode favored over deprecated active field.
+	if !r.Active {
+		// Purposeful breaking change that will be encountered whenever the legacy `active` field was set to false.
+		// This is in order to transition to the use of `mode` field.
+		// If `active` is set to true, this is OK because mode defaults to "ENABLED".
```

Comment:
Nit: this is misleading; active is 100% disabled. The only allowed value is true (which is also the default).
```suggestion
```